aerich/utils.py

```diff
@@ -101,3 +104,43 @@ def import_py_file(file: Union[str, Path]) -> ModuleType:
     module = importlib.util.module_from_spec(spec)  # type:ignore[arg-type]
     spec.loader.exec_module(module)  # type:ignore[union-attr]
     return module
+
+
+def diff_fields(
```

Comment:
`get_dict_diff_by_key` might be a better name because it doesn't really matter if dicts represent fields.